panel/src/components/Views/System/SystemView.vue

```diff
@@ -42,6 +52,30 @@ export default {
 			this.exceptions.map((exception) => console.warn(exception));
 			console.info("End of errors from the update check.");
 		}
+	},
+	methods: {
+		copy() {
+			const info = JSON.stringify(
+				{
+					version: this.environment[1].value,
+					license: this.environment[0].value,
+					php: this.environment[2].value,
+					server: this.environment[3].value,
```

Comment:
This is certainly the best way to avoid sending the data to the frontend twice, but it could easily break if we reorganize the system view. Maybe we could instead send those values as a separate `info` prop from the backend and then add the security and plugin info dynamically in the frontend.